src/components/SelectableList.js

```diff
@@ -1,18 +1,41 @@
-import {useState} from "react";
-import {Card, CardContent, CardHeader, List, ListItem, ListItemText} from "@material-ui/core";
+import {
+    Card,
+    CardActions,
+    CardContent,
+    CardHeader,
+    Icon,
+    IconButton,
+    List,
+    ListItem,
+    ListItemText
+} from "@material-ui/core";
 
-export const SelectableList = (props) => {
+export const SelectableList = props => {
 
     const items = props.items;
     const selectedItem = props.selectedItem;
     const idSelector = props.idSelector;
     const labelSelector = props.labelSelector;
 
+    const addItem = props.onAddItem;
+    const editItem = props.onEditItem;
+    const deleteItem = props.onDeleteItem;
+
     const selectItem = itemId => {
         const item = items.find(item => idSelector(item) === itemId);
         props.onSelect && props.onSelect(item);
     };
 
```

Comment:
très redondant, pourquoi ne pas le gérer simplement avec une liste et itérer dessus ?